[docs] Fix allowedHostPaths description in SecurityPolicy docs (#21642)
deckhouse/deckhouse: modules/015-admission-policy-engine/docs/FAQ.md

1. Create a SecurityPolicy that matches the [baseline](https://kubernetes.io/docs/concepts/security/pod-security-standards/#baseline) or [restricted](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted) policy while also editing the list of `policies` elements as you see fit.
1. Add a label to your namespace that matches the `namespaceSelector` in the SecurityPolicy. In the examples below, the label is `security-policy.deckhouse.io/baseline-enabled: "true"` or `security-policy.deckhouse.io/restricted-enabled: "true"`.

{% alert level="info" %}

> The `allowedHostPaths` field defines the list of allowed path prefixes for hostPath mounts. An empty list (`[]`) forbids hostPath for all paths. If the field is omitted, there are no restrictions.

Reviewer: ```suggestion
The `allowedHostPaths` field defines the list of allowed path prefixes for `hostPath` mounts. An empty list (`[]`) forbids `hostPath` for all paths. If the field is omitted, no restrictions apply.
```
